index.html

```diff
@@ -104,25 +117,51 @@ <h1 style="text-align: center; ">
 community will mirror the information for the popular identifier and setup
 a working redirect to restore service to the rest of the Web.
         </p>
+        <h2 id="new">Creating a new identifier</h2>
         <p>
-If you would like to add or update a permanent identifier to the website, the preferred
+If you would like to add or update a permanent identifier of the form <code>https://w3id.org/...</code>, the preferred
 procedure is to perform the following steps:
         </p>
         <ol>
-          <li><a href="https://github.com/perma-id/w3id.org">Fork the website</a> on Github.</li>
+          <li><em>Fork</em> <a href="https://github.com/perma-id/w3id.org">the <em>Respoitory</em> for this system</a> on Github.</li>
           <li>Add or update a new redirect entry and commit your changes.</li>
-          <li>Submit a pull request for your changes.</li>
+          <li>Submit a <em>Pull Request</em> for your changes.</li>
         </ol>
         <p>
-Please help out the maintainers of the service with the following:
+The maintainers of this system will then action that <em>Pull Requests</em> and merge it into this 
```

Comment:
"will then action that" sounds odd.  Maybe "will then check the".

Reply:
Addressed in an edit below